### PR TITLE
Add Product Recipes Information Architecture Document

### PR DESCRIPTION
**Pull request by tapaswenipathak**
As I am GSoD'19 applicant, prev

- https://github.com/mattermost/mattermost-api-reference/pull/448
- https://github.com/mattermost/docs/pull/2697
- https://github.com/mattermost/docs/pull/2698
- https://github.com/mattermost/docs/pull/2732
- https://github.com/mattermost/docs/pull/2741

### CODE REVIEW

File: source/administration/product-recipes-information-architecture.rst [hunk 1]
```diff
@@ -0,0 +1,23 @@
+Product Recipes Information Architecture
+=========================================
+
+This document defined the product recipes information architecture. The product
```
Comment (amyblais) [inline source/administration/product-recipes-information-architecture.rst]:
```suggestion
This document defines the product recipes information architecture. The product
```

File: source/administration/product-recipes-information-architecture.rst [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+recipes might include information on
```
Comment (amyblais) [inline source/administration/product-recipes-information-architecture.rst]:
```suggestion
recipes might include information on:
```

File: source/administration/product-recipes-information-architecture.rst [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+- Getting user traction for a product
+- Product installation recipes
+- Enablement of KPIs
+- Enabling user requests
+
+Organising Product Recipes
```
Comment (amyblais) [inline source/administration/product-recipes-information-architecture.rst]:
```suggestion
Organizing Product Recipes
```

File: source/administration/product-recipes-information-architecture.rst [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+--------------------------
+
+Defined are the things you should be following for organising product recipes.
```
Comment (amyblais) [inline source/administration/product-recipes-information-architecture.rst]:
```suggestion
Below are the things you should follow for organizing product recipes.
```

File: source/administration/product-recipes-information-architecture.rst [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+
+- Write as per the document present in `administration/product-recipes-template.rst`.
+- You should write all the product recipes in the `product-recipes` directory.
+- If you create any new file you should add the file in the table of contents
+  document.
+- Every product recipe should be linked with blog.
```
Comment (amyblais) [inline source/administration/product-recipes-information-architecture.rst]:
```suggestion
- Every product recipe should be linked with a blog.
```

File: source/administration/product-recipes-information-architecture.rst [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+- Different teams and members can use product recipe created, make sure you
```
Comment (amyblais) [inline source/administration/product-recipes-information-architecture.rst]:
```suggestion
- Different teams and members can use the product recipes that were created. Make sure you
```

File: source/guides/administrator.rst [hunk 7]
```diff
@@ -136,7 +136,8 @@ Learn how to maintain your Mattermost system.
    /administration/health-check*
    /administration/announcement-banner.rst
    /administration/bulk-export.rst
- 
+   /adminstration/product-recipes-information-architecture.rst
```
Comment (amyblais) [inline source/guides/administrator.rst]: See comment here: https://github.com/mattermost/docs/pull/2741#issuecomment-497034900.

These docs are actually internal processes, so a more suitable place for these would be in https://docs.mattermost.com/process/documentation-guidelines.html.

File: source/product-recipes/sample-product-recipes.rst [hunk 8]
```diff
@@ -0,0 +1,19 @@
+Problem Statement
+-----------------
+
+Write the problem statement.
+
+Pre-Installation/Pre-Configuration Notes
+----------------------------------------
+
+Write the pre installation and pre configuration notes.
```
Comment (amyblais) [inline source/product-recipes/sample-product-recipes.rst]:
```suggestion
Write the pre-installation and pre-configuration notes.
```

File: source/product-recipes/sample-product-recipes.rst [hunk 9]
```diff
@@ -0,0 +1,19 @@
...
+
+Instructions Setup and Configuring the Solution
```
Comment (amyblais) [inline source/product-recipes/sample-product-recipes.rst]:
```suggestion
Instructions for Setting Up and Configuring the Solution
```

File: source/product-recipes/sample-product-recipes.rst [hunk 10]
```diff
@@ -0,0 +1,19 @@
...
+-----------------------------------------------
+
+Write the instructions setup and implementation of the solution.
```
Comment (amyblais) [inline source/product-recipes/sample-product-recipes.rst]:
```suggestion
Write the instructions for the setup and implementation of the solution.
```

File: source/product-recipes/sample-product-recipes.rst [hunk 11]
```diff
@@ -0,0 +1,19 @@
+Problem Statement
```
Comment (amyblais) [inline source/product-recipes/sample-product-recipes.rst]: This document appears to be missing a main title. Also this one should be added to the table of contents.

Also, is the purpose of this page to provide example recipes?

Comment (tapaswenipathak) [reply source/product-recipes/sample-product-recipes.rst]: Yes.

**Comment by hanzei** [comment]
@amyblais @wiersgallak Could you please review when you have the time?

**Comment by tapaswenipathak** [comment]
folks, gentle reminder for review.

File: source/process/product-recipes-information-architecture.rst [hunk 12]
```diff
@@ -0,0 +1,23 @@
...
+
+Organizing Product Recipes
+--------------------------
+
+Below are the things you should follow for organizing product recipes.
+
+- Write as per the document present in `administration/product-recipes-template.rst`.
+- You should write all the product recipes in the `product-recipes` directory.
+- If you create any new file you should add the file in the table of contents
+  document.
```
Comment (wiersgallak) [inline source/process/product-recipes-information-architecture.rst]: @tapaswenipathak Can we adjust the language to be more active?  Changing language from "you should..." to "write all products recipes in the 'product-recipes' directory." (one example)

File: source/product-recipes/sample-product-recipes.rst [hunk 13]
```diff
@@ -0,0 +1,24 @@
...
+----------------------------------------
+
+Write the pre-installation and pre-configuration notes.
+
+Instructions for Setting Up and Configuring the Solution
+-----------------------------------------------
+
+Write the instructions for the setup and implementation of the solution.
+
+Defining Recipe Category For Search and Retrieval
```
Comment (wiersgallak) [inline source/product-recipes/sample-product-recipes.rst]: Should this be "Product Recipe Category"

**Comment by wiersgallak** [comment]
@tapaswenipathak Thank you for your work on this document.  Per our discussion this morning, I am going to close this PR, as we think this project needs some additional strategizing and planning to incorporate it into our processes.  We appreciate the work you have put into it and look forward to additional contributions from you in the future. 
cc: @amyblais

**Comment by tapaswenipathak** [comment]
@wiersgallak I would suggest as discussed if you can write on the ticket as well, just for people who might think about writing for the ticket, thanks for the effort from your side as well.

**Comment by tapaswenipathak** [comment]
@wiersgallak I would suggest as discussed if you can write _internal_ on the ticket as well, just for people who might think about writing for the ticket, thanks for the effort from your side as well.

**Pull request closed without merging**